### DISCUSSION TITLE
Questions about time stamps

### DISCUSSION THREAD

**Discussion by Jinbow-0217** [SynxFlow/SynxFlow#23] [Q&A] [answered]
Hi, Dr. Xia. I saw that the input files for the flood simulation: h_BC_1, hU_BC_1, and precipitation_source_all include time information, as well as start time, end time, backup time and output interval that need to be entered when running the program. Are there any dependencies between these times?

**Comment by Colintsl** [comment]
same question too :)

**Comment by xiaxilin** [comment, marked as answer]
Hi, the times in these files all have the start date and they mean the same thing. This is how they relate to each other.

**Reply by Jinbow-0217** [reply]
I see timestamps 0 and 3600 in both h_BC_1 and hU_BC_1, but the start and end times of the input predicted by the model are 0 and 7200. Do the timestamps not need to correspond?

**Reply by xiaxilin** [reply]
No, they don't need to. The model can interpolate or extrapolate.

**Reply by Jinbow-0217** [reply]
Thank you, Dr.Xia.
